test/test_resign.py

```diff
@@ -1,85 +1,125 @@
-import os
 import shutil
 
 from conftest import have_faketime
 from helper import run
+from decode_cms import decode_cms
+
+
+# each signer cert's issuer + serial
+SIGNERS = {
+    "release-1": ("O=Test Org, CN=Test Org Provisioning CA Release", 1),
+    "release-2018": ("O=Test Org, CN=Test Org Provisioning CA Release", 2),
+    "autobuilder-1": ("O=Test Org, CN=Test Org Provisioning CA Development", 1),
+    "autobuilder-2": ("O=Test Org, CN=Test Org Provisioning CA Development", 2),
+}
+
+
+def get_cms(bundle_path):
+    with open(bundle_path, "rb") as bundle_file:
+        bundle_file.seek(-8, 2)
+        cms_length_bytes = bundle_file.read(8)
+        cms_length = int.from_bytes(cms_length_bytes, "big")
+        cms_start = bundle_file.tell() - cms_length - 8
+        bundle_file.seek(cms_start)
+        cms_data = bundle_file.read(cms_length)
```

Comment:
Maybe with some guidance:

```suggestion
        # read CMS/signature size from bundle (8 bytes)
        bundle_file.seek(-8, 2)
        cms_length_bytes = bundle_file.read(8)
        # calculate CMS offset and length
        cms_length = int.from_bytes(cms_length_bytes, "big")
        cms_start = bundle_file.tell() - cms_length - 8
        # read CMS data from bundle file
        bundle_file.seek(cms_start)
        cms_data = bundle_file.read(cms_length)
```